flow360/component/surface_mesh_v2.py

```diff
@@ -165,19 +164,27 @@ def submit(self, description="", progress_callback=None) -> SurfaceMeshV2:
         resp = RestApi(SurfaceMeshInterfaceV2.endpoint).post(req.dict())
         info = SurfaceMeshMetaV2(**resp)
 
-        ##:: upload geometry files
+        ##:: upload surface mesh file
         surface_mesh = SurfaceMeshV2(info.id)
-        heartbeat_info = {"resourceId": info.id, "resourceType": "Geometry", "stop": False}
+        heartbeat_info = {"resourceId": info.id, "resourceType": "SurfaceMesh", "stop": False}
         # Keep posting the heartbeat to keep server patient about uploading.
         heartbeat_thread = threading.Thread(target=post_upload_heartbeat, args=(heartbeat_info,))
         heartbeat_thread.start()
 
+        surface_mesh._webapi._upload_file(
```

Comment:
upload the surface mesh first, then upload mapbc file if it exists. It is more intuitive.